Makefile

```diff
@@ -211,7 +211,7 @@ KIND_CLUSTER_NAME ?= "integration-tests"
 INTEGRATION_TEST_TIMEOUT ?= "45m"
 E2E_TEST_TIMEOUT ?= "45m"
 KONG_CONTROLLER_FEATURE_GATES ?= GatewayAlpha=true
-GOTESTFMT_CMD ?= $(GOTESTFMT) -hide successful-downloads,empty-packages -showteststatus
+export GOTESTSUM_FORMAT=standard-verbose
```

Comment:
What's the difference between a regular makefile assignment and this?

Reply:
Without the `export`, we have to inject the env variable explicitly to every command, like the following: [839f341](https://github.com/Kong/kubernetes-ingress-controller/pull/3021/commits/839f341a75e7cac7c5a005b01fec1ef2ac8de7ab)

I think that will be less confusing 👍